db/migrate/20170916194350_create_medical_statuses.rb

```diff
@@ -0,0 +1,10 @@
+class CreateMedicalStatuses < ActiveRecord::Migration[5.1]
+  def change
+    create_table :medical_statuses do |t|
+      t.string :name
+      t.string :description
```

Comment:
You want a text type for description fields: common gotcha, strings are 255 chars max. You'll need to create a new migration with a change_column call.